ansible/roles/ansible-tvault-contego-extension/tasks/object_store_service.yml

```diff
@@ -12,11 +12,32 @@
       - python-s3fuse-plugin-cent7
     state: latest
   when: >
-      ((ansible_distribution_major_version=="7" and ansible_distribution== centos and PYTHON_VERSION=="python2") or
+      ((ansible_distribution_major_version=="7" and ansible_distribution=="CentOS" and PYTHON_VERSION=="python2") or
```

Comment:
just a suggestion, better to compare by converting lowercase

Reply:
thanks, changes done